doc/plv8.md

```diff
@@ -459,13 +461,20 @@ PL/v8 provides the following utility built-in functions.
 - `WARNING`
 - `ERROR`
 
+```javascript
+var msg = 'world';
```

Comment:
unfortunately, this won't work - we're published on pgxn, which does not support gfm.  please check how other examples are formatted.

Reply:
Ahh, sorry I was only thinking about github syntax.  I'll clean that up right now.

Reply:
don't worry, you're not alone :) it was @theory who put in the PR to fix the previous formatting to work correctly with pgxn.  thanks much for the update, I'll finish the review in just a little bit!